src/components/Profile/AddPasskey.tsx

```diff
@@ -189,6 +192,11 @@ const AddPasskey = ({ responseMessages }: { 	responseMessages: (value: IResponse
 		} else {
 			setProfile();
 		}
+		const platform = navigator.platform.toLowerCase();
+        console.log("platform", platform)
+        if (platform.includes('linux')) {
```

Comment:
Don't use static value create ENUM for device type

Reply:
Resolved